mediatype.go

```diff
@@ -1,9 +0,0 @@
-package smithy
-
-// The MediaType interface is intended to be implemented by string and
-// byte array types whose values can be defined by RFC638 media types.
-type MediaType interface {
-	// Describes the contents of the string or byte array using a media type
-	// as defined by RFC6838.
-	MediaType() string
-}
```

Comment:
Intentional?

Reply:
yeah this type wasn't used at all and was an artifact that was never deleted from an old design.